tests/helpers.bash

```diff
@@ -2,6 +2,7 @@
 
 BUILDAH_BINARY=${BUILDAH_BINARY:-$(dirname ${BASH_SOURCE})/../bin/buildah}
 IMGTYPE_BINARY=${IMGTYPE_BINARY:-$(dirname ${BASH_SOURCE})/../bin/imgtype}
+COPY_BINARY=${COPY_BINARY:-$(dirname ${BASH_SOURCE})/../bin/copy}
```

Comment:
I know this follows the `XXX_BINARY` convention, but at this point I'd like to suggest breaking away from that and using `BUILDAH_COPY_HELPER` instead. Something that makes it obvious and explicit that this is a custom buildah thing.

Reply:
Could you open a separate PR to change IMGTYPE and COPY to the name you want.

Reply:
>Could you open a separate PR to change IMGTYPE and COPY to the name you want.

It's too late for IMGTYPE: that change would need to be coordinated across gating tests on RHEL and Fedora, on all possible branches. That's just not sane.

I was hoping we could get this one done properly right now, so we could get things clean in gating tests.